mava/systems/ippo/networks.py

```diff
@@ -187,6 +188,8 @@ def make_discrete_networks(
     policy_layer_sizes: Sequence[int],
     critic_layer_sizes: Sequence[int],
     observation_network: Callable = utils.batch_concat,
+    activate_mlp_final: bool = False,
```

Comment:
Should we not always set it to True? Do we need this flag?

Reply:
Totally agree. I just made a flag for now to be able to benchmark this again. I'll remove this flag :)

Reply:
[These benchmarks](https://www.notion.so/instadeep/Experiments-52346a5de10d444583896c8d5b026b25?p=3f347755c06147d1922c4cbd235fca19&pm=s) show conclusively that we should activate these intermediate layers so I'll set it to be the default for moving forward and remove the flag.

Reply:
Great :partying_face: